Jupyter FileExtractionTask task fails

```
2021-02-25 17:17:55 [INFO] Execution of [['sudo', 'image_export.py', '--logfile', '/evidence/1614273470-4a7632dfe76c428797ce0ddeafc684bf-FileArtifactExtractionTask/4a7632dfe76c428797ce0ddeafc684bf.log', '-w', '/evidence/1614273470-4a7632dfe76c428797ce0ddeafc684bf-FileArtifactExtractionTask/export', '--partitions', 'all', '--artifact_filters', 'JupyterConfigFile', '-d', '/dev/loop1']] failed with status 1
2021-02-25 17:17:55 [INFO] image_export.py failed for artifact JupyterConfigFile.
```

https://github.com/google/turbinia/pull/766/checks?check_run_id=1981113701#step:9:842

Still failing.
```
2021-06-03 13:50:49 [ERROR] JupyterAnalysisTask Task failed with exception: [write() argument must be str, not bytes]
2021-06-03 13:50:49 [ERROR] Traceback (most recent call last):
  File "/usr/local/lib/python3.6/dist-packages/turbinia-20210602.dev22+gb5f5613-py3.6.egg/turbinia/workers/__init__.py", line 916, in run_wrapper
    self.result = self.run(evidence, self.result)
  File "/usr/local/lib/python3.6/dist-packages/turbinia-20210602.dev22+gb5f5613-py3.6.egg/turbinia/workers/analysis/jupyter.py", line 65, in run
    fh.write(output_evidence.text_data.encode('utf8'))
TypeError: write() argument must be str, not bytes

2021-06-03 13:50:49 [ERROR] JupyterAnalysisTask Task failed with exception: [write() argument must be str, not bytes]
2021-06-03 13:50:49 [INFO] Traceback (most recent call last):
  File "/usr/local/lib/python3.6/dist-packages/turbinia-20210602.dev22+gb5f5613-py3.6.egg/turbinia/workers/__init__.py", line 916, in run_wrapper
    self.result = self.run(evidence, self.result)
  File "/usr/local/lib/python3.6/dist-packages/turbinia-20210602.dev22+gb5f5613-py3.6.egg/turbinia/workers/analysis/jupyter.py", line 65, in run
    fh.write(output_evidence.text_data.encode('utf8'))
TypeError: write() argument must be str, not bytes
```

--- a/turbinia/workers/analysis/jupyter.py
+++ b/turbinia/workers/analysis/jupyter.py
@@ -61,9 +61,9 @@ class JupyterAnalysisTask(TurbiniaTask):
     result.report_data = report
 
     # Write the report to the output file.
-    with open(output_file_path, 'w') as fh:
-      fh.write(output_evidence.text_data.encode('utf8'))
-      fh.write('\n'.encode('utf8'))
+    with open(output_file_path, 'w', encoding='utf-8') as fh:
+      fh.write(output_evidence.text_data)
+      fh.write('\n')
 
     # Add the resulting evidence to the result object.
     result.add_evidence(output_evidence, evidence.config)

